.gitignore

```diff
@@ -1 +1,4 @@
 /target
+
+swp
+**/*.*.swp
```

Comment:
I prefer keeping per-user configs like this in global git config files and only add things exclusive to the repo in its .gitignore. You can configure a global .gitignore file like this in .gitconfig:
```
[core]
	excludesFile = ~/.gitignore
```